bin/web/views/index.ejs

```diff
@@ -30,6 +30,13 @@
 </head>
 <body>
 <form id="form" action="">
+  <select id="model">
+    <!-- options: 7B, 13B, 30B, 65B -->
+    <option value="7B">7B</option>
+    <option value="13B">13B</option>
+    <option value="30B">30B</option>
+    <option value="65B">65B</option>
+  </select>
```

Comment:
Really useful, and works fine.  
Might consider adding a  text input box for # of threads, and temperature too.   I'm finding the model pretty sensitive to temperature settings.